pkg/testcase/localpathstorage.go

```diff
@@ -30,11 +30,12 @@ func TestLocalPathProvisionerStorage(deleteWorkload bool) {
 	Eventually(func(g Gomega) {
 		var res string
 		fmt.Println("Writing and reading data from pod")
```

Comment:
Do we need this Print statement? If yes can we update with the proper actions? It looks like it is just reading data from pod...